pkg/handlers/index/alias_es.go

```diff
@@ -0,0 +1,191 @@
+package index
+
+import (
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog/log"
+	"github.com/zinclabs/zinc/pkg/core"
+	"github.com/zinclabs/zinc/pkg/meta"
+	"github.com/zinclabs/zinc/pkg/zutils"
+)
+
+type Alias struct {
+	Actions []Action `json:"actions"`
+}
+
+type Action struct {
+	Add    *base `json:"add"`
+	Remove *base `json:"remove"`
+}
+
+type base struct {
+	Index   string   `json:"index"`
+	Alias   string   `json:"alias"`
+	Indices []string `json:"indices"`
+	Aliases []string `json:"aliases"`
+}
+
+func AddOrRemoveESAlias(c *gin.Context) {
```

Comment:
Can you provide documentation comments please.

Reply:
done